universal-application-tool-0.0.1/conf/messages.en-US

```diff
@@ -45,6 +45,19 @@ content.benefits=Get benefits
 content.description1=CiviForm lets you apply for many benefits at once by reusing information.
 content.description2=Get started by choosing an application below.
 
+error.not_found_title=We''re sorry, we lost
+error.not_found_title_end=the page
+error.not_found_description_a=Don''t worry though, the benefits
+error.not_found_description_a_end=are still there.
+error.not_found_description_b=Find them on
+error.not_found_description_b_end=the
+error.not_found_description_link=homepage
```

Comment:
issue (blocking): please reword to "We were unable to find the page you tried to visit. Please go back or return to the homepage."